html2asketch/model/text.js

```diff
@@ -29,7 +29,7 @@ class Text extends Base {
     obj.text = this._text;
     obj.style = this._style.toJSON();
 
-    obj.resizingConstraint = 47;
+    obj.resizingConstraint = this.setResizingConstraints('height');
```

Comment:
This is why I made the method return too as not everything is on `this`.

Reply:
Actually based on other discussion I'll just use the new `calculateResizingContraintValues` method here